apps/src/aichat/chatApi.ts

```diff
@@ -7,17 +7,19 @@ import {
 import HttpClient from '@cdo/apps/util/HttpClient';
 import {CHAT_COMPLETION_URL} from './constants';
 import Lab2Registry from '../lab2/Lab2Registry';
+import {TutorTypes} from '../aiTutor/types';
```

Comment:
nit: I would name the enum `TutorType`, similar to the singular you used for other enums (`Role`, `Status`)